extensions/ql-vscode/src/vscode-tests/no-workspace/contextual/astBuilder.test.ts

```diff
@@ -65,6 +65,35 @@ describe('AstBuilder', () => {
     )).to.deep.eq(expectedRoots);
   });
 
+  it('should build the AST children', async () => {
+    // just test one of the children to make sure that the structure is right
+    const astBuilder = createAstBuilder();
+    const roots = await astBuilder.getRoots();
+
+    expect(roots[0].children[1].parent).to.eq(roots[0]);
+    // break the recursion
+    (roots[0].children[1] as any).parent = undefined;
+    (roots[0].children[1] as any).children = undefined;
+
+    const child = {
+      children: undefined,
+      fileLocation: undefined,
+      id: 26367,
+      label: 'body: [Block] { ... }',
```

Comment:
Do we have a test for a target without an edge label?

Reply:
I will add something.